CHANGES.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 
 
 - Add @@clear-and-update-index-server-index. [ksuess]
+- Add profiles for testing single lingual and multi lingual. [ksuess]
```

Comment:
I would say "single lingual: is not the right term. I tend to "monolingual". But I asked for help here, my English is not good enough to decide on this.

Reply:
@ksuess Sorry for my pickiness, but there is no such term as `single lingual`. According to native speakers both, `single-language` or `monolingual`, are fine. Additional `monolingual`  and `multilingual` are written as one word.

Reply:
If it's only this… That's easy to change. :-)